src/lib/PostgrestQueryBuilder.ts

```diff
@@ -65,20 +77,23 @@ export default class PostgrestQueryBuilder<T> extends PostgrestBuilder<T> {
    * Performs an UPDATE on the table.
    *
    * @param values  The values to update.
+   * @param returning By default the updated record is returned. Set this to 'minimal' if you don't need this value.
    */
-  update(values: Partial<T>): PostgrestFilterBuilder<T> {
+  update(values: Partial<T>, { returning = 'representation' } = {}): PostgrestFilterBuilder<T> {
     this.method = 'PATCH'
-    this.headers['Prefer'] = 'return=representation'
+    this.headers['Prefer'] = `return=${returning}`
     this.body = values
     return new PostgrestFilterBuilder(this)
   }
 
   /**
    * Performs a DELETE on the table.
+   *
+   * @param returnDeleted  If `true`, return the deleted row(s) in the response.
    */
-  delete(): PostgrestFilterBuilder<T> {
+  delete({ returnDeleted = true } = {}): PostgrestFilterBuilder<T> {
```

Comment:
@soedirgo I think this one should be `{ returning = 'representation' }`. Similar to the other functions.
```suggestion
  delete({ returning = 'representation' } = {}): PostgrestFilterBuilder<T> {
```